backtrack/src/app/blacklist/blacklist.page.html

```diff
@@ -0,0 +1,24 @@
+<ion-header>
+    <ion-toolbar>
+        <ion-title>
+              Blacklist
+      </ion-title>
+      <ion-buttons slot="start">
+        <ion-back-button></ion-back-button>
+      </ion-buttons>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content>
+    <ion-card class="tracker-list">
+      <ion-card-content>
+        <ng-container *ngIf="trackers$ | async as trackers">
```

Comment:
don't think you need to do this asynchronously- you should be able to just do line 16 without this